### PR TITLE
[ID] Update `Beatmap/Online_offset` and `Beatmap/Guest_difficulty`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by daud-ahn — COMMENTED** [review]
can the image be updated with the new website?

File: wiki/Beatmap/Online_offset/id.md [hunk 1]
```diff
@@ -10,10 +8,10 @@ tags:
...
 
-Ini dapat diubah oleh anggota dari [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team), dan beberapa pengguna seperti [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) dan [Developer](/wiki/People/The_Team/Global_Moderation_Team). Artinya, online offset tidak dapat dilihat oleh pengguna biasa pada situs web, namun dapat dilihat di dalam gim. Inilah tampilan yang diperlihatkan kepada pengguna yang diperbolehkan untuk melihat dan mengubahnya:
+Ini dapat diubah oleh anggota dari [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team), dan beberapa pengguna seperti [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) dan [Developer](/wiki/People/The_Team/Global_Moderation_Team). Artinya, online offset tidak dapat dilihat oleh pengguna biasa pada situs web, namun dapat dilihat di dalam permainan. Berikut adalah tampilan yang diperlihatkan kepada pengguna yang memiliki akses untuk melihat dan mengubahnya:
```
Comment (daud-ahn) [inline wiki/Beatmap/Online_offset/id.md:13]: GMT is developer?

Comment (ZeroPyrozen) [reply wiki/Beatmap/Online_offset/id.md:13]: I will change that after I get clarification on which team member can edit online offset. I've asked BN, they are sure only NAT can change online offset.

**Comment by ZeroPyrozen** [comment]
> can the image be updated with the new website?

Online offset feature still [work in progress](https://github.com/ppy/osu-web/pull/7474) in new website?

**Comment by daud-ahn** [comment]
> > can the image be updated with the new website?
> 
> Online offset feature still [work in progress](https://github.com/ppy/osu-web/pull/7474) in new website @daud-ahn

oh.. i see

**Comment by Walavouchey** [comment]
is this good to go?

**Comment by ZeroPyrozen** [comment]
yeah, this is good to go

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**